common/rabbitmq/templates/deployment.yaml

```diff
@@ -71,8 +71,8 @@ spec:
         {{- end }}
         {{- if .Values.readinessProbe.enabled }}
         readinessProbe:
-          exec:
-            command: ["rabbitmqctl", "node_health_check"]
+          tcpSocket:
+            port: {{ default "5672" .Values.ports.public }}
```

Comment:
I would go here with the name `"public"`, as it avoids having to change it in multiple places.

Reply:
Sure: You can refer to the port by its number and then it should be the same value as in the [port definition](https://github.com/sapcc/helm-charts/pull/3687/files/410b8b4dcc17b2ef6556a1e9dae208be30b11631#diff-41bc547432263860bb872966d794df676bc675faeac1a21bb3e7e68155c2d5b4R85-R86), or you can refer to it by the name given in the same definition.

My personal preference would be to assign a name in the port-definition and refer everywhere else in kubernetes to it by its name.

Reply:
OK i think got it, you mean using the port definition, which is defined in the container ports, correct ?
```
 tcpSocket:
      port: public
```

Reply:
Exactly

Reply:
Fixed!